pkg/discovery/dtofactory/pod_entity_dto_builder.go

```diff
@@ -287,6 +292,15 @@ func (builder *podEntityDTOBuilder) getPodCommoditiesBoughtFromQuota(quotaUID st
 		}
 		commoditiesBought = append(commoditiesBought, commBought)
 	}
+
+	// Access commodity.
+	accessComm, err := sdkbuilder.NewCommodityDTOBuilder(proto.CommodityDTO_VMPM_ACCESS).
```

Comment:
Earlier in the code you already have:
				entityDTOBuilder.IsMovable(proto.EntityDTO_NAMESPACE, false)
Do we still need the access commodity here?